tests/integration/services/test_tei_integration.py

```
@@ -0,0 +1,341 @@
+            "مرحبا بالعالم",
+            "こんにちは世界",
+        ]
+
+        result = embed._get_text_embeddings(texts)
+
+        assert len(result) == 5
+        assert all(len(emb) == 1024 for emb in result)
+        # All should be valid embeddings
+        assert all(any(x != 0.0 for x in emb) for emb in result)
+
+    def test_very_long_text(self, check_tei_services: None) -> None:
```

Ruff (0.14.0)

Unused method argument: `check_tei_services`

(ARG002)